utils/python/CIME/XML/GenericXML.py

```diff
@@ -84,11 +84,16 @@ def get_value(self, item):
         get_value is expected to be defined by the derived classes, if you get here it is an error.
         """
         logging.debug("Get Value for "+item)
+        result = None
         if item in self.lookups.keys():
-            return self.lookups[item] 
+            result = self.lookups[item]
         if item in os.environ:
-            return os.environ.get(item)
-#        expect(False, "Not implemented")
+            result = os.environ.get(item)
+
```

Comment:
Hmm, didn't anticipate that usecase. Well, I'm fine with deleting the warning for now.